### DISCUSSION TITLE
CANNOT FIND A WAY TO CLEAR THE REMIX AI ASSISTANT FROM THE REMIX IDE

### DISCUSSION THREAD

**Discussion by workrate99** [Cyfrin/foundry-full-course-cu#2859] [Q&A]
good morning from here, i just started the  solidity smart contract developer course , i hop onto remix ide, but now the remix AI assistant section is taking a huge chunk of my space on the ide, making it hard for me to code and i can't see any close or minimize button to clear it away. Please is there a way to remove it.

**Comment by EngrPips** [comment]
Share your screen so we can understand what you are talking about.

**Comment by LordAdhaar** [comment]
![image](https://github.com/user-attachments/assets/3e1890e6-4bf7-460c-9bf4-548f799978e3)
I don't know exactly what your issue is since as @EngrPips said it would be helpful if you share a screenshot 
But you can click on those two down arrows to close it

**Reply by EngrPips** [reply]
Interesting.
